### PR TITLE
Implement navigation bar and set up page routing

### PR DESCRIPTION
**Pull request by Haichen-Wang**
## Overview

Implement the navigation bar in React with responsive design.

Use react router to transition to corresponding pages with navigation bar interactions.

## Changes Made

- Implement navigation bar in `NavigationBar.js`
- Add js files for each page, and set them up
- Refactor App.js using react router
- Add responsive design to navigation bar so that it looks great on mobile

## Test Coverage

Tested using `npm start` in chrome.

## Screenshot

https://user-images.githubusercontent.com/55666780/114311828-e6e6a280-9b22-11eb-99af-5827ebebd130.mov

https://user-images.githubusercontent.com/55666780/114311837-ef3edd80-9b22-11eb-82ad-b9a0650a0ed7.mov

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,77 +1,33 @@
...
+import CourseTopicCard from "./Components/CourseTopicCard";
+import "./Components/styles.css";
+import NavigationBar from "./Components/NavigationBar";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Apply from "./Pages/Apply";
+import Apps from "./Pages/Apps";
+import Courses from "./Pages/Courses";
+import Home from "./Pages/Home";
+import Support from "./Pages/Support";
+import Team from "./Pages/Team";
```
Comment (aliciaxw) [inline src/App.js:14]: we should probably start organizing our imports - how does splitting imports into two sections (each alphabetized) sound? 

section 1: third party code (`react-router-dom`)
section 2: local imports (everything else like `./Pages`

File: src/Pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+export default function Home() {
+    return (
+        <div className="App">
+            <header className="App-header">
+                <CourseTopicCard
+                    icon={sunIcon}
+                    intro="PRODUCT THINKING"
+                    header="Designing the right thing"
+                    description="Learn how to find the right problem space, do real user reserach, and ideate a variety of solutions using industry processes."
+                    img={mockup}
```
Comment (aliciaxw) [inline src/Pages/Home.js:18]: it seems like the font family broke for the course topic card? perhaps a css change

**Review by aliciaxw — APPROVED** [review]
amazing work!

File: src/App.js (hunk 1 above)
Comment (Haichen-Wang) [reply src/App.js:14]: Great idea! Not organizing the imports is probably fine when the number of imports is small, but once it gets large, we definitely should be doing that.

File: src/Pages/Home.js (hunk 2 above)
Comment (Haichen-Wang) [reply src/Pages/Home.js:18]: Good catch! I'll look into that.

**Pull request merged**